src/components/Sidebar/Sidebar.js

```diff
@@ -69,6 +69,14 @@ const INTRO_SEQUENCE_STEPS = [
   'add-projects-fade-in',
 ];
 
+// Creating a portal to get around an issue with position fixed and transform property
+// which creates a weird offset while dragging
+// Ref https://github.com/atlassian/react-beautiful-dnd/issues/499
+let portal: HTMLElement = document.createElement('div');
```

Comment:
I'll check later if this is fixing the problem. I think it should be OK.
I'm wondering if we should move the portal to it's own location e.g. add it to `App` and reference it here by class or id. I haven't used it before but I think it could be useful for other components too. E.g. modals. Or would the modal also dynamically add a div to body?
I think it's OK to add it dynamically here but if we're using portals also in other places we should have a common style how we're adding them.